tests/expected/function-contract/simple_replace_pass.expected

```diff
@@ -1,4 +1,4 @@
-div.assertion\
+.assertion\
```

Comment:
why are we missing the name of the functions here?

Reply:
because the name now includes a closure. Something like `div.{closure#1}.assertion`

Reply:
btw, I just didn't want to include the closure name here because it's auto generated and I'm not sure how stable that id is.